web/src/main/java/org/cbioportal/genome_nexus/web/MyVariantInfoController.java

```diff
@@ -32,7 +32,7 @@ public void MyVaraintInfoController(MyVariantInfoService myVariantInfoService)
 
     @ApiOperation(value = "Retrieves myvariant information for the provided list of variants",
         nickname = "fetchMyVariantInfoAnnotationGET")
-    @RequestMapping(value = "/my_variant_info/{variant:.+}",
+    @RequestMapping(value = "/my_variant_info/variant/{variant:.+}",
```

Comment:
changed the url to be the same as myvariant.info, in case we want to add more of their endpoints